module/hashes.go

```diff
@@ -0,0 +1,8 @@
+package module
+
+// Hashes contains hexadecimal checksums of code libraries brought in by running a Build
+type Hashes struct {
```

Comment:
Why are we computing all of these? Isn't there just one hash that's relevant per dependency?